action.yml

```diff
@@ -6,35 +6,44 @@ branding:
   color: 'orange'
 
 inputs:
```

Comment:
Would be sweet if this auto-generated the docs page.

Reply:
Our docs are based on Markdown, and live in a different repository. It's definitely possible, but not really worth the effort and complexity. We don't change options all that often.